src/scripts/main.js

```diff
@@ -21,7 +21,34 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  for (const key1 in food) {
+    const ol = document.createElement('ol');
+
+    ol.textContent = key1;
+    document.body.appendChild(ol);
```

Comment:
Instead of appending `ol` to `document.body`, it should be appended to the `element` parameter to properly use the function's arguments.